sprint/switch_branch.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# This script allows switching drupal branch, for example, from 8.9.x to 9.0.x
+# or back
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+if [[ $# != 1 ]]; then
+  echo "Please provide a branch to switch to. For example 'switch_branch.sh 9.0.x'"
+  exit 1
+fi
+
+
+target_branch=$1
+
+pushd drupal8
+set -x
+ddev exec "git stash && git reset --hard && git checkout ${target_branch} && git pull && git stash pop"
+ddev composer require drush/drush:^10
```

Comment:
Will drush 10 be incompatible when downgrading from 9.0.x to 8.8.x?

Reply:
According to [compatibility chart](http://docs.drush.org/en/master/install/#drupal-compatibility) drush 10 is fine with Drupal 8.4+